dom-renderables/DOMElement.js

```diff
@@ -204,7 +204,7 @@ DOMElement.prototype.onDismount = function onDismount() {
  * @return {undefined} undefined
  */
 DOMElement.prototype.onShow = function onShow() {
-    this.setProperty('display', 'block');
+    if (this._node.isShown()) this.setProperty('display', 'block');
```

Comment:
> Currently, once a parent node#show called, the parent node calls all the children's node#onShow and child nodes` DOMElement#onShow although some child nodes are manually hidden which should not be affected by a parent.

Good point. I doubt there is a good solution to this in the current architecture. If a parent node is hidden, all its children are hidden (`this._shown === false`). I think that's an assumption we have to make and I don't think we want to change that.

What I was suggesting (seems like I misunderstood the intention behind this PR), is that the node's show state should be updated in the Dispatch, not in the Node's `show` method.